e3sm-unified.def

```diff
@@ -11,7 +11,7 @@ From: continuumio/miniconda
 
 %post -c /bin/bash
     . /opt/conda/etc/profile.d/conda.sh
-    conda create -n e3sm-unified-nox -c conda-forge -c e3sm -c cdat/label/v81 python=2.7 e3sm-unified=1.2.6 mesalib
+    conda create -n e3sm-unified-nox -c conda-forge -c e3sm -c cdat/label/v82 python=3.7 e3sm-unified=1.3.0 mesalib
```

Comment:
@xylar The base image for this container is [miniconda](https://hub.docker.com/r/continuumio/miniconda). Can you try changing it to [miniconda3](https://hub.docker.com/r/continuumio/miniconda3)?
From this
```
From: continuumio/miniconda
```
to this
```
From: continuumio/miniconda3
```